src/ose.js

```diff
@@ -134,7 +134,7 @@ Hooks.on("renderSidebarTab", async (object, html) => {
     const styling =
       "border:none;margin-right:2px;vertical-align:middle;margin-bottom:5px";
     $(
-      `<button data-action="userguide"><img src='systems/ose/assets/dragon.png' width='16' height='16' style='${styling}'/>Old School Guide</button>`
+      `<button data-action="userguide"><img src='${OSE.assetsPath}/dragon.png' width='16' height='16' style='${styling}'/>Old School Guide</button>`
```

Comment:
While you're here, could you add `type="button"` to this button? While not explicitly necessary (an unset button type is equivalent to a button of type "submit" and I don't think this button is in a form), I think it'll be good to model good behavior for when we start into the new sheets.

Reply:
I see that this has been added